ajax/libs/overpass/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "overpass",
+  "version": "3.0.2",
+  "filename": "overpass-webfont/overpass.css",
```

Comment:
We need minified overpass-webfont/overpass.cssPlease. Namely overpass-webfont/overpass.min.css. And please use `overpass-webfont/overpass.min.css` as `filename`.